src/Microsoft.OData.Core/ODataBatchOperationRequestMessage.cs

```diff
@@ -55,17 +64,19 @@ public sealed class ODataBatchOperationRequestMessage : IODataRequestMessage, IO
             IODataBatchOperationListener operationListener,
             string contentId,
             IODataUrlResolver urlResolver,
-            bool writing)
+            bool writing,
+            IList<string> dependsOnRequestIds = null)

```

Comment:
> = null [](start = 45, length = 7)

since the constructor is private, you should just always provide a value when you call the method, rather than define a default for the parameter.